src/main/java/uk/gov/justice/hmpps/prison/api/model/AssessmentDetail.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
```

Comment:
Did you mean to commit the comments?

Reply:
I was been consistent?! But it isn't useful so removing